src/__test__/components/data-exploration/generic-gene-table/LaunchPathwayAnalysisModal.test.jsx

```diff
@@ -37,10 +44,14 @@ getDiffExprGenes.mockImplementation(() => async () => Promise.resolve(genesList)
 describe('Pathway analysis modal ', () => {
   beforeEach(() => {
     jest.clearAllMocks();
+    usePantherDBSpecies.mockImplementation(() => ({
```

Comment:
I think you should mock the fetch here, right? I don't know if the swr thing is mocked when you mock the json fetch)

Reply:
replaced using mocked fetch